Extension/MultipathUpgrade/Enum/EquipmentProperties.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace YAPO.MultipathUpgrade.Enum
+{
+    [Flags]
+    public enum EquipmentProperties
```

Comment:
What about throwing weapons? May be a nice addition when there is a need for skirmishers

Reply:
I don't think the game is currently in a stage where people will want to decide between infantry with or without throwing weapons. besides, most infantry that is any good has some available. You make a valid point where I think people will want to prefer polearms over one handed swords to counter cavalry. However adding this much logic to it, it might be better to just enable the player to add troops to a favorite list having at least x% in their army of those.